src/main/java/hudson/plugins/git/browser/GitRepositoryBrowser.java

```diff
@@ -117,5 +119,27 @@ public static URL encodeURL(URL url) throws IOException {
         }
     }
 
+    public static boolean initialChecksAndReturnOk(Item project, String cleanUrl){
```

Comment:
I would think you want this to be protected rather than public.  Do you intend that others can call this method that are not extending this class?
```suggestion
    protected static boolean initialChecksAndReturnOk(Item project, String cleanUrl){
```

Reply:
Of course yes, my bad, I have fixed it.